package.json

```diff
@@ -2,32 +2,34 @@
   "name": "aries-framework",
   "version": "0.0.0",
   "license": "Apache-2.0",
-  "main": "build/src/index.js",
-  "types": "build/src/index.d.ts",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
   "files": [
-    "build/src"
+    "build"
   ],
   "repository": {
     "url": "https://github.com/hyperledger/aries-framework-javascript",
     "type": "git"
   },
   "scripts": {
-    "compile": "tsc",
+    "compile": "tsc --project tsconfig.build.json",
+    "check-types:build": "yarn compile --noEmit",
+    "check-types": "tsc --noEmit",
     "lint": "eslint .",
     "prettier": "prettier --ignore-path .gitignore '**/*.+(js|json|ts|md|yml|yaml)'",
     "format": "yarn prettier --write",
     "check-format": "yarn prettier --list-different",
     "test": "jest --verbose",
     "dev": "ts-node-dev --respawn --transpile-only ./tests/mediator.ts",
-    "prod:start": "node ./build/tests/mediator.js",
-    "prod:start-ws": "node ./build/tests/mediator-ws.js",
-    "prod:build": "rm -rf build && yarn compile",
-    "validate": "npm-run-all --parallel lint compile",
+    "mediator:start": "ts-node ./tests/mediator.ts",
+    "mediator:start-ws": "ts-node ./tests/mediator-ws.ts",
+    "validate": "npm-run-all --parallel lint check-types",
     "prepack": "rm -rf build && yarn compile",
     "release": "release-it",
     "prepare": "husky install"
   },
   "dependencies": {
+    "@types/ws": "^7.4.5",
```

Comment:
Is it in regular `dependencies` with a purpose?

Reply:
It was. But I removed it again. We were directly importing the type,  but not the package and therefor it couldn't find the type. However the dependency on this type package also introduces a dependency on `@types/node` which clashes with `@types/react-native`. Very annoying